notebooks/network_analysis_sec.ipynb

```diff
@@ -0,0 +1,523 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ax = heating_stats.plot.bar(\n",
+    "    title='The demand of various heating subsectors',\n",
+    "    xlabel=\"\"\n",
+    ")\n",
+    "\n",
+    "ax.set_xticklabels(heating_stats.index.get_level_values('carrier'), rotation=45, ha=\"right\")\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "How is heating energy supplied (heat pump, district heating, resistive heater, gas boiler)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[274.9255349451954, 0.2902017678572082, 27.878132885439577]"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "heating_supply_stats = n.statistics.supply(comps=[\"Link\"], aggregate_time=\"mean\").loc[\n",
+    "    lambda x: x.index.get_level_values('carrier').str.contains('heat|boiler|pump|resistive', case=False)\n",
+    "]\n",
+    "\n",
+    "heating_types = ['Gas Boiler', 'Heat Pump', 'Resistive Heater']\n",
```

Comment:
What if the network includes further heating types? Maybe it does not include it in this example, but maybe in future there are more? Either go for a more comprehensive technology selection, or at least print a warning like "only the technologies xxx are considered here". Otherwise, it might lead to confusions